src/components/ScoreInput.js

```diff
@@ -18,11 +27,10 @@ const ScoreInput = React.createClass({
     return (
       <div className="scope-input">
         <div className="score-block scope-label">Score your <mark>{this.props.scope}</mark> work.</div>
-        <input className="score-block score-picker" id={this.props.scope + '_score'}
-          type="range" min="1" max="5" onChange={this.updateScoreLabel} />
-        <label className="score-block score-picker-label" id={this.props.scope + '_label'}>OK</label>
+        <input ref="scorePicker" type="range" min="1" max="5" onChange={this.update} className="score-block score-picker" />
+        <label id={this.props.scope + '_label'} className="score-block score-picker-label">{sliderValues[3]}</label>
```

Comment:
that id was being used for `document.getElementById(labelId).textContent` label updates, which I'm replacing w/a ref.